### PR TITLE
Create db install script

### PR DESCRIPTION
**Pull request by jpa198**

### CODE REVIEW

**Comment by jpa198** [comment]
Don't actually merge this, as we want to use it as the base of a feature branch for the entire refactor

File: shared_workflow/shared_automated_workflow.py [hunk 1]
```diff
@@ -184,3 +187,57 @@ def check_mgmt_queue(queue_entries: List[str], run_name: str, proc_type: int, lo
...
     return False
+
+
+def parse_fsf(fault_selection_file: str):
+    """Parses a fault selection file into a list of (fault, realisation count) pairs"""
+    faults = []
+    with open(fault_selection_file) as fault_file:
+        for line in fault_file.readlines():
+            fault, count, *_ = line.split()
+            if count[-1] == 'r':
```
Comment (jasonmotha) [inline shared_workflow/shared_automated_workflow.py]: Not the only current realisation count specifier. This possibly will be refactored out in the future.

File: shared_workflow/shared_automated_workflow.py [hunk 2]
```diff
@@ -184,3 +187,57 @@ def check_mgmt_queue(queue_entries: List[str], run_name: str, proc_type: int, lo
...
+                count = int(count[:-1])
+            else:
+                count = int(count)
+            faults.append((fault, count))
+    return faults
+
+
+def parse_config_file(config_file_location: str, logger: Logger = workflow_logger.get_basic_logger()):
+    """Takes in the location of a wrapper config file and creates the tasks to be run.
+    Requires that the file contains the keys 'run_all_tasks' and 'run_some', even if they are empty
```
Comment (jasonmotha) [inline shared_workflow/shared_automated_workflow.py]: update comment

File: scripts/cybershake/install_cybershake_fault.py [hunk 3]
```diff
@@ -144,8 +141,6 @@ def install_fault(fault_name, n_rel, root_folder, version, stat_file_path, seed=
 
         vm_params_dict.update(vm_add_params_dict)
 
-        create_mgmt_db.create_mgmt_db([], sim_struct.get_mgmt_db(root_folder), srf_files=srf)
```
[the remark this answers is not in the document]
Comment (jpa198) [reply scripts/cybershake/install_cybershake_fault.py]: After the installation refactor these lines can go back in, as the refactor won't use this function

File: e2e_tests/queue_monitor_tests.py [hunk 4]
```diff
@@ -75,9 +75,9 @@ def setup(self):
 
     def install(self):
         print("Installing database")
-        create_mgmt_db.create_mgmt_db(
-            ["{}_REL{:0>2}".format(self.realisation_name, i) for i in range(1, 1 + self.realisations)],
-            sim_struct.get_mgmt_db(self.stage_dir),
+        scripts.management.install_mgmt_db.create_mgmt_db_from_faults(
+            [(self.realisation_name, self.realisations)],
```
[the remark this answers is not in the document]
Comment (jpa198) [reply e2e_tests/queue_monitor_tests.py]: Done

File: e2e_tests/e2e_config_simple_18p5p3p4.json [hunk 5]
```diff
@@ -1,8 +1,8 @@
 {
   "test_dir": "/nesi/nobackup/nesi00213/RunFolder/EndToEndTest/tests/",
-  "data_dir": "/nesi/project/nesi00213/EndToEndTest/test_data_simple_18p5p3p4/Data",
-  "fault_list": "/nesi/project/nesi00213/EndToEndTest/test_data_simple_18p5p3p4/list.txt",
-  "bench_dir": "/nesi/project/nesi00213/EndToEndTest/test_data_simple_18p5p3p4/benchmark",
+  "data_dir": "/nesi/project/nesi00213/EndToEndTest/test_data_simple_18p5p3p4_new/Data",
+  "fault_list": "/nesi/project/nesi00213/EndToEndTest/test_data_simple_18p5p3p4_new/list.txt",
+  "bench_dir": "/nesi/project/nesi00213/EndToEndTest/test_data_simple_18p5p3p4_new/benchmark",
```
[the remark this answers is not in the document]
Comment (jpa198) [reply e2e_tests/e2e_config_simple_18p5p3p4.json]: As discussed

**Pull request closed without merging**